Build.cmd

```diff
@@ -19,12 +19,12 @@ SET CMDHOME=%~dp0
 @REM Remove trailing backslash \
 set CMDHOME=%CMDHOME:~0,-1%
 
-set SPARKCLR_HOME=%CMDHOME%\run
+set SPARKCLR_HOME="%CMDHOME%\run"
```

Comment:
With Windows cmd script, nothing should break in a set command from whitespace in variables on right-hand-side.

In general, the usual guideline is only to add the quotes at point-of-use - which looks like Line 27 below -- and I think you end up adding two sets of quotes with changes in 22+27 and 22+108?